mappings/net/minecraft/world/gen/foliage/FoliagePlacer.mapping

```diff
@@ -26,9 +47,14 @@ CLASS net/minecraft/class_4647 net/minecraft/world/gen/foliage/FoliagePlacer
 		ARG 5 leaves
 	METHOD method_23451 isInvalidForLeaves (Ljava/util/Random;IIIII)Z
 		ARG 2 baseHeight
-		ARG 3 x
-		ARG 4 y
-		ARG 5 z
+		ARG 3 dx
+		ARG 4 dy
+		ARG 5 dz
```

Comment:
why was this changed?

Reply:
because this is not xyz but relative xyz from the tree root

Reply:
ok